src/main/resources/view/HelpWindow.fxml

```diff
@@ -14,6 +15,10 @@
   </icons>
   <scene>
     <Scene>
+        <stylesheets>
```

Comment:
The indentation here seems off

Reply:
It appears that the indentation for this file was inconsistent to begin with. Some sections are indented with 2 spaces, some with 3, and this change with 4 spaces. Since it's out of scope for this PR, perhaps the change can be standardized to 2 spaces similar to `MainWindow.fxml` and the rest of the file can be fixed in a separate PR?

Reply:
my thought when highlighting this is that since we are inserting something, at least the inserted portion should be formatted correctly on its own, so the open and closing tags should be at the same indentation level, currently the opening tag is misaligned with the closing tag with an offset of 1.